premerge/libcxx_runners_values.yaml

```diff
@@ -0,0 +1,47 @@
+githubConfigUrl: "https://github.com/llvm"
+githubConfigSecret: "github-token"
+
+minRunners: 0
+maxRunners: 8
```

Comment:
This can get bumped up to 32 as well.

Here there is a possibility that we run into some issues if both the new runner set and the older runner set try to spawn a total of more than 16 jobs at the same time (pods would queue), but theoretically things should just queue. In practice we might run into some minor issues based on what I've observed, but nothing major, and the switchovers occur pretty rarely anyways.